build.rs

```diff
@@ -546,6 +546,7 @@ fn cc(
     if target.os != "none"
         && target.os != "redox"
         && target.os != "windows"
+        && target.os != "freebsd"
```

Comment:
If this only affects i686 then there should be a `target.arch` check here too, right?

Reply:
seems not a ring issue. I'll reopen if it still related to ring.